test/framework/src/main/java/org/elasticsearch/transport/nio/AcceptorEventHandler.java

```diff
@@ -82,8 +83,9 @@ public void acceptChannel(NioServerSocketChannel nioServerChannel) throws IOExce
      * @param nioServerChannel that accepting a connection
      * @param exception that occurred
      */
-    public void acceptException(NioServerSocketChannel nioServerChannel, Exception exception) {
-        logger.debug("exception while accepting new channel", exception);
+    void acceptException(NioServerSocketChannel nioServerChannel, Exception exception) {
+        logger.debug(new ParameterizedMessage("exception while accepting new channel from server channel: {}",
```

Comment:
Since this is at the debug level, should you use a supplier here to avoid instantiating the parameterized message if not needed?